### PR TITLE
Document how to set Kafka Topic config options

### PR DESCRIPTION
**Pull request by Cali0707**
This documents how to set Kafka topic config options on a broker.

## Proposed Changes 

- Add a paragraph explaining that we support all kafka topic config options, and how to set them

Note: this should not merge until after https://github.com/knative-extensions/eventing-kafka-broker/pull/3671

### CODE REVIEW

File: docs/eventing/brokers/broker-types/kafka-broker/README.md [hunk 1]
```diff
@@ -113,6 +113,27 @@ different `name` on your Kafka Broker's `spec.config` field.
...
+  # Number of topic partitions
+  default.topic.partitions: "10"
+  # Replication factor of topic messages.
+  default.topic.replication.factor: "3"
+  # A comma separated list of bootstrap servers. (It can be in or out the k8s cluster)
+  bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092"
+  # Here is our retention.ms config
+  default.topic.config.retention.ms: "3600"
+```
+
```
Comment (pierDipi) [inline docs/eventing/brokers/broker-types/kafka-broker/README.md:135]: Can we link to where to find the option names in the Kafka docs website ?

Comment (Cali0707) [reply docs/eventing/brokers/broker-types/kafka-broker/README.md:135]: Added

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Comment by Cali0707** [comment]
/unhold
The feature is merged now

**Pull request merged**